packages/web3-provider/src/subproviders/EthGasStationProvider.js

```diff
@@ -23,7 +23,7 @@ class EthGasStationProvider extends SubProvider {
     const jason = await res.json()
     if (typeof jason[GAS_PRICE_KEY] !== 'undefined') {
       // values come from EGS as tenths of gwei
-      return String(jason[GAS_PRICE_KEY] * 1e8)
+      return '0x' + (jason[GAS_PRICE_KEY] * 1e8).toString(16)
```

Comment:
That won't break the relayer, will it ?

Reply:
tests passed

Reply:
I feel like I should elaborate that standad JSON-RPC interfaces return in hex.  This shouldn't break anything, tests passed, but it is a valid concern and we should test it when it gets deployed.